docs/pydantic.md

```diff
@@ -445,10 +445,47 @@ for example:
 
 ## Adding Custom Field-level Attributes
 
-To add `custom field attributes` the `metadata` attribute must be set in `pydantic.Field`. For more info check [adding-custom-field-level-attributes](https://marcosschroh.github.io/dataclasses-avroschema/fields_specification/#adding-custom-field-level-attributes) section for `dataclasses`.
+Custom Field-level attributes can be added by leveraging the Pydantic's [`Field`](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field) function, with the [json_schema_extra](https://docs.pydantic.dev/latest/concepts/json_schema/#customizing-json-schema) property, ensuring you have the `metadata` attribute enclosing the custom field-level attributes you want to add.
+
+Let's say you have the following model:
+
+```python
+from dataclasses_avroschema.pydantic import AvroBaseModel
+from pydantic import Field
+
+class Person(AvroBaseModel):
+    first_name: str = Field(json_schema_extra={"metadata": {"description": "This is the person's first name"}}, default=None)
+    last_name: str = None
+    phone: str = None
+
+print(Person.avro_schema())%
```

Comment:
The `%` should not be there

Reply:
@marcosschroh apologies for that! Removed it